src/libsystemd/sd-bus/bus-control.c

```diff
@@ -713,7 +713,7 @@ _public_ int sd_bus_get_name_creds(
                 }
 
                 r = bus_creds_add_more(c, mask, pid, 0);
-                if (r < 0)
+                if (r < 0 && r != -ESRCH) /* If the process is gone by now, then don't augment, return what we got */
```

Comment:
I'd prefer to have the comment reversed: "Return the error, but ignore ESRCH which just means the process is already gone" ...